spec/flows/asylum_and_immigration_flag_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+RSpec.describe "Asylum and immigration flag", :controlled_flag, :asylum_and_immigration_flag, type: :feature do
+  it "adds a new matter type screen" do
+    start_assessment
+    fill_in_provider_screen
+    fill_in_level_of_help_screen
+    fill_in_matter_type_screen(choice: "Another legal matter")
+    confirm_screen("applicant")
```

Comment:
There's lots of duplication here. Could it be extracted into common methods or before blocks?